Jenkinsfile

```diff
@@ -28,8 +28,6 @@ pipeline {
                     agent { label 's61113u16 (litecore)' }
                     environment {
                        BRANCH = "${BRANCH_NAME}"
-                       CC = "gcc-7"
-                       CXX = "g++-7"
```

Comment:
To double check that these are not needed anymore and not accidentally deleted.

Reply:
Correct, the backing machine was upgraded and will fail if these are present